maestro-ai/src/main/java/maestro/ai/Prediction.kt

```diff
@@ -270,4 +270,4 @@ object Prediction {
         return response.text ?: ""
     }
 
-}
+}
```

Comment:
some of these styles and changes were related to inteliJs linter, but if I need to adjust something feel free to comment 🙏